stream-chat-android/build.gradle

```diff
@@ -78,6 +78,8 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 dependencies {
     api project(":stream-chat-android-offline")
 
+    implementation Dependencies.androidxStartup
+    api Dependencies.threeTenAbp
```

Comment:
Since we use the threeten backport internally, I had to include it as an API dependency since `DateFormatter` accepts a `org.threeten.bp.LocalDateTime`. Do we foresee any issues doing this? Should we abstract this somehow? 🤔